lib/active_fulfillment/service.rb

```diff
@@ -25,15 +25,73 @@ def valid_credentials?
       true
     end
 
-    # API Requirements for Implementors
+    # Fulfill an order.
+    #
+    # @example
+    #
+    #     carrier = ActiveFulfillment::Carrier.new(...)
+    #     response = carrier.fulfill(order.id, order.shipping_address, order.line_items)
+    #     assert response.success?
+    #
+    #
+    # @param order_id [String] Unique identifier for the order
+    # @param shipping_address [?] The address to ship the order to
```

Comment:
Is this a String, Hash, or a composite object? We should document this, and potentially add a basic Address object that people can use or mimic using duck typing.